tests/system/action/motion/test_update.py

```diff
@@ -310,3 +348,82 @@ def test_only_motion_allowed(self) -> None:
         assert "Found assignment/1 but only motion is allowed." in response.json.get(
             "message", ""
         )
+
+    def test_update_no_permissions(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion.update",
+            {
+                "id": 111,
+                "title": "title_bDFsWtKL",
+                "text": "text_eNPkDVuq",
+                "reason": "reason_ukWqADfE",
+            },
+        )
+
+    def test_update_permission(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion.update",
+            {
+                "id": 111,
+                "title": "title_bDFsWtKL",
+                "text": "text_eNPkDVuq",
+                "reason": "reason_ukWqADfE",
+            },
+            Permissions.Motion.CAN_MANAGE,
+        )
+
+    def test_update_permission_metadata_no_wl(self) -> None:
+        self.create_meeting()
+        self.user_id = self.create_user("user")
+        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [3])
+        self.set_group_permissions(3, [Permissions.Motion.CAN_MANAGE_METADATA])
+        self.set_models(self.permission_test_model)
+        response = self.request(
+            "motion.update",
+            {
+                "id": 111,
+                "title": "title_bDFsWtKL",
+                "text": "text_eNPkDVuq",
+                "reason": "reason_ukWqADfE",
+            },
+        )
+        self.assert_status_code(response, 403)
+        assert "Forbidden fields:" in response.json["message"]
+
+    def test_update_permission_metadata_and_wl(self) -> None:
+        self.create_meeting()
+        self.user_id = self.create_user("user")
+        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [3])
+        self.set_group_permissions(3, [Permissions.Motion.CAN_MANAGE_METADATA])
+        self.set_models(self.permission_test_model)
+        self.set_models({"motion_category/2": {"meeting_id": 1}})
+        response = self.request(
+            "motion.update",
+            {
+                "id": 111,
+                "category_id": 2,
+            },
+        )
+        self.assert_status_code(response, 200)
+
+    def test_update_permission_submitter_and_wl(self) -> None:
+        self.create_meeting()
+        self.user_id = self.create_user("user")
+        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [3])
+        self.permission_test_model["motion_submitter/1"]["user_id"] = self.user_id
+        self.set_models(self.permission_test_model)
+        response = self.request(
+            "motion.update",
+            {
+                "id": 111,
+                "title": "title_bDFsWtKL",
+                "text": "text_eNPkDVuq",
+                "reason": "reason_ukWqADfE",
+            },
+        )
+        self.assert_status_code(response, 200)
```

Comment:
yes, the combined test of metadata and submitter is missing here.

Reply:
Add a test.